### DISCUSSION TITLE
Using turbo pack and turbo repo and webpack project that does not have next js.

### DISCUSSION THREAD

**Discussion by erangamapa** [vercel/turborepo#3003] [Help] [answered]
I would like to know if i am able to use the build and hot reload features of turbo pack and turbo repo without converting project to Next js Project.

Thanks

**Comment by anthonyshew** [comment, marked as answer]
Turbopack is being purpose-built for Next.js to prove out its core concepts and is planned to expand to a general purpose bundler in the future!

You can use Turborepo independently of Turbopack. Turborepo is framework agnostic.
